task-2-SERVLETS/Self-check.md

```diff
@@ -0,0 +1,13 @@
+1. What is the Servlet Container? List some of them.
+2. What is the life-cycle of a servlet?
+3. What are life-cycle methods for a servlet do you know and what is their functions?
+4. What is REST?
+5. What HTTP methods are used according to the REST?
```

Comment:
I suppose this would be better: 
5. Which HTTP methods are used according to the REST?

Reply:
```suggestion
5. Which HTTP methods are used according to the REST?
```